src/month.rs

```diff
@@ -161,6 +161,22 @@ impl Month {
             Month::December => "December",
         }
     }
+
+    ///     Get the length of the month in a given year
+    pub const fn length(&self, year: i32) -> u8 {
```

Comment:
Not sure about this name. `len()` would be more idiomatic in Rust but maybe it should be a little more specific like `num_days()`?

Reply:
I think length is a pretty straightforward explanation of what the function does. I didn't like how num_days() sounds

Reply:
I disagree -- `len()`/`length()` is not very specific and not fully intuitive on `Month` IMO -- for example, it could yield a `Duration` instead. We already use the `num_days()` for a number of methods in chrono and it would be a good fit here.